salesforce/canvas/views/main_chat.ejs

```diff
@@ -46,6 +46,8 @@
             'https://www.gstatic.com/agent-assist-ui-modules/staging/agent_coaching.js',
           smart_reply:
             'https://www.gstatic.com/agent-assist-ui-modules/staging/smart_reply.js',
+          live_translation:
+            'https://www.gstatic.com/agent-assist-ui-modules/staging/live_translation.js',
```

Comment:
You may need to add an entry to the prod list as well, otherwise the loadScript function might break